uw-frame-components/portal/notifications/partials/notifications-full.html

```diff
@@ -1,41 +1,99 @@
-														 ng-click="button.action"
-														 target="{{button.target}}"
-														 aria-label="{{button.label}}">
-										{{button.label}}
-									</md-button>
-								</div>
-					    </a>
-
-							<div class="action" ng-click='dismiss(notification)' ng-hide="!notification.dismissable || isDismissed(notification)"  layout="row" layout-align="center center">
-	            	<i class='fa fa-times'tooltip="Dismiss notification" tooltip-placement="top" tooltip-popup-delay="300" tooltip-trigger="mouseenter"></i>
-							</div>
-              <div class="action" ng-click="undismiss(notification)" ng-hide="!(mode === 'old')">
-                 <i class='fa fa-undo'tooltip="Revert Dismiss of notification" tooltip-placement="left" tooltip-popup-delay="300" tooltip-trigger="mouseenter"></i>
-              </div>
-        		</li>
-        		<li class="notification-item read" ng-if="isEmpty || (countWithDismissed() === 0 && mode === 'new') || (mode === 'old' && dismissedCount() === 0)">You have no <span ng-show='dismissedCount() === 0 && countWithDismissed() !== 0'>dismissed</span> notifications at this time.</li>
-        	</ul>
-		</div>
-	</div>
+    <md-tabs md-dynamic-height md-border-bottom>
+      <md-tab aria-label="click to see new notifications" label="New">
+        <md-content>
+          <md-list class="notifications-list" flex>
+            <md-list-item class="secondary-button-padding notification-item"
+                          ng-class="{ priority: notification.priority }"
+                          ng-repeat="notification in notifications | orderBy:['-priority', '-id']"
+                          ng-if="!isDismissed(notification)"
+                          layout="row"
+                          layout-align="start center">
+              <!-- NOTIFICATION TEXT AND ACTION BUTTONS -->
+              <a class="md-list-item-text"
+                 ng-href="{{ notification.actionURL }}"
+                 target="_blank"
+                 aria-label="{{ notification.actionAlt }}"
+                 layout="row"
+                 layout-align="start center"
+                 layout-xs="column"
+                 layout-align-xs="center start">
+                <span><i class="fa fa-exclamation-triangle" ng-if="notification.priority"></i> {{ notification.title }}</span>
+                <div ng-if="notification.actionButtons && notification.actionButtons.length > 0" class="notification-buttons">
+                  <md-button class="md-raised"
+                             ng-href="{{button.url}}"
+                             ng-repeat="button in notification.actionButtons track by button.label"
+                             ng-class="{'md-primary' : $index === 0, 'md-accent' : $index > notification.buttonText.length / 2 - 1}"
+                             ng-click="button.action"
+                             target="{{button.target}}"
+                             aria-label="{{button.label}}">
```

Comment:
Is this `aria-label` necessary when it's the same text as the body of the `<md-button>` element? Peeking a docs, looks like maybe only need `aria-label` for icon buttons that don't have natural label text?